symplyphysics/laws/thermodynamics/heat_transfer_equation.py

```diff
@@ -0,0 +1,52 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+# Fourier's law Q = k * (T1-T2)/ d * S
+# where:
+# Q = heat flow 
+# k = material thermal conductivity, ability of a material to transfer heat
+# T1 = inside temperature
+# T2 = outside temperature
+# d = cross section thickness
+# S = cross-sectional area
+# 
+# The Fourier formula is used to determine the amount of heat flow that is transferred through a material from one side to the other due to the material's thermal conductivity, temperature gradient, thickness and cross-sectional area. 
```

Comment:
`determine the amount of heat flow` - but you determine not amount of heat flow in this formula, but rate of heat flow.